config/locales/en/document_types.yml

```diff
@@ -2,7 +2,7 @@ en:
   document_types:
     default:
       fields:
-        title:
+        title_and_base_path:
```

Comment:
I think this renaming could be moved to a separate commit, where the argument for renaming can be made 👍

Reply:
Oops! I realise I didn't offer any explanation in the commit. I think it's important to have the change in this commit, since it provides a little motivation for it - I've updated the commit message.